overlord/snapstate/handlers.go

```diff
@@ -1250,6 +1244,38 @@ func (m *SnapManager) undoCopySnapData(t *state.Task, _ *tomb.Tomb) error {
 	return nil
 }
 
+// writeMigrationStatus is used to re-write the state and sequence file (if they
+// exist). This must be called after the migration undo procedure is done
+// since it only then we know the actual final state of the migration.
```

Comment:
nitpick: the "it" seems to be a leftover

Reply:
It is, thank you